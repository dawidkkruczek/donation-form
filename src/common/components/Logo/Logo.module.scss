.root {
  display: block;
}

.logo {
  fill: #423c66;
  max-height: 100%;
  width: 100%;
}
