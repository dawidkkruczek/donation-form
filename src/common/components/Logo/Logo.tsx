import React, { ReactElement } from 'react';
import classNames from 'classnames';
import { ReactComponent as LogoComponent } from './assets/logo.svg';
import styles from './Logo.module.scss';

export interface Props {
  className?: string;
}

function Logo({ className }: Props): ReactElement {
  return (
    <a className={classNames(styles.root, className)} href="/">
      <LogoComponent className={styles.logo} />
    </a>
  );
}

export default Logo;
