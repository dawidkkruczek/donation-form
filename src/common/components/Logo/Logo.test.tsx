import React from 'react';
import { render } from '@testing-library/react';
import Logo from './Logo';

describe('common/components/Logo', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Logo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
