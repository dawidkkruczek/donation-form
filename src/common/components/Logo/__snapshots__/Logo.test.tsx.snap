// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/components/Logo should render correctly 1`] = `
<DocumentFragment>
  <a
    class="root"
    href="/"
  >
    <svg
      class="logo"
    >
      Logo.svg
    </svg>
  </a>
</DocumentFragment>
`;
